client/state/selectors/index.js

```diff
@@ -134,6 +133,7 @@ export hasBrokenSiteUserConnection from './has-broken-site-user-connection';
 export hasInitializedSites from './has-initialized-sites';
 export hasSiteComments from './has-site-comments';
 export hasUnsavedUserSettings from './has-unsaved-user-settings';
+export hasUserAskedADirectlyQuestion from './has-user-asked-a-directly-question';
```

Comment:
I love that you fixed the order of selectors, but could we have done that in a separate PR?